plugins/CoreAdminHome/vue/src/JsTrackingCodeGenerator/JsTrackingCodeGenerator.vue

```diff
@@ -0,0 +1,618 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <ContentBlock
+    anchor="javaScriptTracking"
+    :content-title="translate('CoreAdminHome_JavaScriptTracking')"
+  >
+    <div id="js-code-options">
+      <p>
+        {{ translate('CoreAdminHome_JSTrackingIntro1') }}
+        <br/><br/>
+        {{ translate('CoreAdminHome_JSTrackingIntro2') }}
+        <span v-html="jsTrackingIntro3a"></span>
```

Comment:
There seems to be a space missing between both texts now.
```suggestion
        <span v-html="jsTrackingIntro3a"></span>&nbsp;
```